specviz/plugins/line_labels/line_labels_plotter.py

```diff
@@ -406,13 +468,6 @@ def _destroy_zoom_markers_thread(self):
 
 class ZoomMarkersThread(QThread):
```

Comment:
This class will need a simple docstring.

Reply:
Done.